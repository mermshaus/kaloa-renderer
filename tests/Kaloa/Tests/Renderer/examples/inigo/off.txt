[off][b]this will not be converted[/b][/off]

A paragraph.

[off]
[ul]
[li]Not a.[/li]
[li]paragraph.[/li]
[/ul]
[/off]

Foo

[noparse][off][/off][/noparse]

[off][noparse][/noparse][/off]
