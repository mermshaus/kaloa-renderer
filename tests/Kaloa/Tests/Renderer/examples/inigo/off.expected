<p>[b]this will not be converted[/b]</p>

<p>A paragraph.</p>

<p><br />
[ul]<br />
[li]Not a.[/li]<br />
[li]paragraph.[/li]<br />
[/ul]</p>

<p>Foo</p>

<p>[off][/off]</p>

<p>[noparse][/noparse]</p>
